packages/circuit-ui/components/CardList/CardList.js

```diff
@@ -24,6 +24,8 @@ const Wrapper = styled(Card)`
   label: cardlist__wrapper;
 
   padding: 0;
+  border-radius: ${(p) => p.theme.borderRadius.mega};
+  border: ${(p) => p.theme.borderWidth.kilo} solid ${(p) => p.theme.colors.n300};
```

Comment:
Saved 2 chars 🎉 (just kidding, this is fine)

```suggestion
  border: ${(p) => `${p.theme.borderWidth.kilo} solid ${p.theme.colors.n300}`};
```